e2e/tests/visual/notification.visual.spec.js

```diff
@@ -0,0 +1,61 @@
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+/**
+ * This test is dedicated to test notification banner functionality and its accessibility attributes.
+ */
+
+const { test, expect } = require('../../pluginFixtures');
+const percySnapshot = require('@percy/playwright');
+
+test.describe('Visual - Check Notification Info Banner of \'Save successful\'', () => {
+    test.beforeEach(async ({ page }) => {
+        // Go to baseURL and Hide Tree
+        await page.goto('./', { waitUntil: 'networkidle' });
+    });
+
+    test('Create a clock, click on \'Save successful\' banner and dismiss it', async ({ page }) => {
+        // Click on button that has text "Create" text
+        await page.locator('button:has-text("Create")').click();
+        // Click on "Clock" option from the "menuitem" list
+        await page.locator('li[role="menuitem"]:has-text("Clock")').click();
+        // Click on button that has text=OK
+        await page.locator('text=OK').click();
+        // Verify there is a button with aria-label="Review 1 Notification"
+        expect(await page.locator('button[aria-label="Review 1 Notification"]').isVisible()).toBe(true);
+        // Verify there is a button with aria-label="Clear all notifications"
+        expect(await page.locator('button[aria-label="Clear all notifications"]').isVisible()).toBe(true);
+        // Click on the div with role="alert" that has "Save successful" text
+        await page.locator('div[role="alert"]:has-text("Save successful")').click();
+        // Verify there is a div with role="dialog"
+        expect(await page.locator('div[role="dialog"]').isVisible()).toBe(true);
+        // Verify the div with role="dialog" contains text "Save successful"
+        expect(await page.locator('div[role="dialog"]').innerText()).toContain('Save successful');
+        await percySnapshot(page, 'Notification banner');
+        // Verify there is a text "Dismiss" visible
+        expect(await page.locator('text=Dismiss').isVisible()).toBe(true);
```

Comment:
Plain `text=` selectors have a high probability of matching more than one selector, which fails the test. Let's make this selector more deterministic by including more selection criteria. Something like `'button:has-text("Dismiss")'` should work.

Reply:
Done! ✅